pages/events/AttendingEvents.js

```diff
@@ -0,0 +1,22 @@
+import { useSession } from "next-auth/react";
+import EventList from "@/components/EventList";
+import { useRouter } from "next/router";
+import Button from "@/components/Button";
+
+export default function AttendingEvents() {
+  const router = useRouter();
+
+  const { data: session, status } = useSession();
+
+  if (status === "unauthenticated") {
+    router.push("/login");
+  }
+
+  return (
+    <>
+      <h1>Attending Events</h1>
```

Comment:
same. please keep the semantics in mind!

Reply:
@julialoeschel Same